package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "devDependencies": {
     "depcheck": "1.3.1",
-    "eslint": "7.32.0",
+    "eslint": "^8.56.0",
```

Comment:
OOC, why this version of eslint? Is it just the latest version from 2022?

Reply:
Good question.  V7 didn't seem to have support for 2022, and V9 introduced some fairly large changes to the config files.  It dropped support for the `.eslintignore`, renamed `eslintrc.js` to `eslint.config.js`, and significantly changed the format of `eslint.config.js`.